docs/README.md

```diff
@@ -33,7 +33,7 @@ There are several optional parameters that could be passed into `CreateVolumeReq
 |-----------------------------|-----------------------------------|----------|---------------------|
 | "csi.storage.k8s.io/fsType" | xfs, ext2, ext3, ext4             | ext4     | File system type that will be formatted during volume creation |
 | "type"                      | io1, io2, gp2, sc1, st1,standard  | gp2      | EBS volume type     |
-| "iopsPerGB"                 |                                   |          | I/O operations per second per GiB. Required when io1 or io2 volume type is specified |
+| "iopsPerGB"                 |                                   |          | I/O operations per second per GiB. Required when io1 or io2 volume type is specified. If this value multiplied by the size of a requested volume produces a value below the minimum or above the maximum IOPs allowed for the volume type, AWS will return an error and volume creation will fail |
```

Comment:
We should probably add a link to the limits: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html

Funnily enough I don't see the minimum IOPS on the chart.

Reply:
added a link.

Yeah it's buried below "io1 and io2 volumes can range in size from 4 GiB to 16 TiB. You can provision from 100 IOPS ..."